lib/matplotlib/tests/test_legend.py

```diff
@@ -544,12 +545,43 @@ def test_window_extent_cached_renderer():
     leg2.get_window_extent()
 
 
-def test_legend_title_fontsize():
+def test_legend_title_fontprop_fontsize():
     # test the title_fontsize kwarg
-    fig, ax = plt.subplots()
-    ax.plot(range(10))
-    leg = ax.legend(title='Aardvark', title_fontsize=22)
-    assert leg.get_title().get_fontsize() == 22
+    with pytest.raises(ValueError):
+        plt.plot(range(10))
+        plt.legend(title='Aardvark', title_fontsize=22,
+                   title_fontproperties={'family': 'serif', 'size': 22})
+
+    plt.plot(range(10))
```

Comment:
Only the `legend` raises here:
```suggestion
    plt.plot(range(10))
    with pytest.raises(ValueError):
        plt.legend(title='Aardvark', title_fontsize=22,
                   title_fontproperties={'family': 'serif', 'size': 22})
```